src/engine/world/world.ts

```diff
@@ -335,25 +343,28 @@ export class World {
      * @param distance The maximum distance to search for Players.
      * @param instanceId The player's active instance.
      */
-    public findNearbyPlayers(position: Position, distance: number, instanceId: string = null): Player[] {
+    public findNearbyPlayers(position: Position, distance: number, instanceId: string | null = null): Player[] {
+        // TODO (Jameskmonger) how should this handle `instanceId` param if it's null?
```

Comment:
I would assume we should probably just return an empty Array

Reply:
I have stopped the param being `null` entirely 😄